.c9/.nakignore

```diff
@@ -0,0 +1,17 @@
+*~backup-*
+.c9revisions
+.c9
+.git
+.svn
+.DS_Store
+.bzr
+.cdv
+~.dep
+~.dot
+~.nib
+~.plst
+.hg
+.pc
+*.min.js
+.nakignore
+/dev
```

Comment:
We don't need to commit this folder to git. You will have to remove it from git and add the folder `.c9` to .gitignore file